test/unittests/serviceRoutines/statisticsTreat_test.cpp

```diff
@@ -64,10 +65,9 @@ TEST(statisticsTreat, delete)
   out       = restService(&ci, rs);
   EXPECT_STREQ(expectedBuf, out.c_str());
 
-  EXPECT_EQ("OK", testDataFromFile(expectedBuf, sizeof(expectedBuf), outfile2)) << "Error getting test data from '" << outfile2 << "'";
   ci.outFormat = JSON;
   out       = restService(&ci, rs);
-  EXPECT_STREQ(expectedBuf, out.c_str());
+  EXPECT_STREQ("{\"message\":\"All statistics counter reset\"}", out.c_str());
```

Comment:
Sure, it is a step in the wrong direction, but not a very important one ...

NTC